modules/webm/SCsub

```diff
@@ -4,7 +4,9 @@ Import("env")
 Import("env_modules")
 
 env_webm = env_modules.Clone()
-
+if env["shared"]:
```

Comment:
webm had compilation issues when converting it to shared build, so instead of spending too much time investigating it remains static in the shared build so gains from shared build can begin to be reaped. I will investigate and fix in another PR.

Reply:
this module is built shared in the latest commit